client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -184,13 +147,12 @@ class ListItem extends PureComponent {
       data: { _id: itemId, name },
       match: {
         params: { id: listId }
-      },
-      socket
+      }
     } = this.props;
 
-    this.itemBusy();
+    this.lockItem();
 
-    return archiveItem(listId, itemId, name, socket);
+    return archiveItem(listId, itemId, name);
```

Comment:
no need to unlock item here?

Reply:
nope, because it's unmounted after archive action so there is nothing to unlock.